src/Symfony/Component/Console/Application.php

```diff
@@ -1091,6 +1091,35 @@ public function setDefaultCommand($commandName, $isSingleCommand = false)
         return $this;
     }
 
+    /**
+     * Defines the "APP_ENV" and "APP_DEBUG" environment variables by consuming --env and --no-debug from the command line arguments.
+     */
+    public static function bootstrapEnv(array &$argv)
```

Comment:
i think adding it to `Symfony\Bundle\FrameworkBundle\Console\Application` makes more sense, the logic seems tied to the framework stack; not the component.

`FWBundle\Application` is already the used class in bin/console.. so should be fine no?

Reply:
Adding it to the component allows standardizing the practice. And the code has not relationships with the bundle. My vote is to keep it in the component.

Reply:
Right :+1: seems fair.

Reply:
but ok to put in FWB if more ppl think it's a better fit :)